wp-api.js

```diff
@@ -224,7 +224,7 @@
 		{
 			idAttribute: 'id',
 
-			urlRoot: WP_API_Settings.root + '/users',
+			urlRoot: WP_API_Settings.root + 'wp/v2/users',
```

Comment:
The API i've settled on for kadamwhite/wordpress-rest-api, which admittedly uses a very different paradigm, is to expose `.namespace` and `.version` methods, then to make individual native types like "posts" automatically call `.namespace('wp').version('v2')`; inasmuch as this is basically the same as refactoring this `wp/v2` string out into a property, as is being discussed here, I'm +1 on making that change and making it something subclasses could override. Won't hold up a merge here, though; I'm going to push the button.